### PR TITLE
[ERSSUP-67731]-[]-[Test file for new test utils]-[DMW]

### PR DESCRIPTION
**Pull request by nhsd-david-wass**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -214,46 +255,37 @@ async def app_restricted_app(
...
+    client, app_restricted_app, _jwt_keys, environment, oauth_url
+):
+    credentials = app_restricted_app["credentials"][0]
+
+    config = ClientCredentialsConfig(
+        environment=environment,
+        identity_service_base_url=oauth_url,
+        client_id=credentials["consumerKey"],
+        jwt_private_key=_jwt_keys["private_key_pem"],
+        jwt_kid="test-1",
```
Comment (nhsd-david-wass) [inline tests/conftest.py:283]: Check that _jwt_keys is okay - is there a parent fixture that I should be using

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me, just a few minor questions/comments 🙂

File: sandbox/package-lock.json
[the remark this answers is not in the document]
Comment (nhsd-david-wass) [reply sandbox/package-lock.json:1]: Yes

File: package-lock.json
[the remark this answers is not in the document]
Comment (nhsd-david-wass) [reply package-lock.json:1]: Yes

**Review by nhsd-jack-wainwright — APPROVED** [review]

**Pull request merged**